src/Program.cs

```diff
@@ -121,10 +116,7 @@ protected override void OnStartupNextInstance(int processId, string[] args)
                         }
                         if (!mainWindow.IsActive)
                         {
-                            if (!Leayal.Shared.Windows.UnmanagedWindowsHelper.SetForegroundWindow(mainWindow))
-                            {
-                                mainWindow.Activate();
-                            }
+                            mainWindow.Activate();
```

Comment:
This is the only place where functionality may be impacted. If needed, I can change it to only use `SetForegroundWindow` on Windows